### DISCUSSION TITLE
CFI Linux

### DISCUSSION THREAD

**Discussion by Landiks** [Divested-Mobile/DivestOS-Build#111] [Q&A] [answered]
I saw a post on Reddit.

Why is CFI not enabled for all devices? Any plans for it to come to more devices?

**Comment by SkewedZeppelin** [comment, marked as answer]
You can't just flip the toggle.

Many require changes to even successfully compile. And even if they do, they might not boot.
If I push out an update that doesn't boot and your device is locked you're forced to wipe your data.

**Comment by rsssxd** [comment]
If there is a test build for Davinci I can try and let you know if it works.


You can upload the test build to a different platform and let me test it. It would be nice to know if it works.
